kword-plugin/src/main/groovy/com/mirego/kword/KWordEnumGenerate.groovy

```diff
@@ -47,6 +47,7 @@ class KWordEnumGenerate extends DefaultTask {
 
     private writeFile(ClassName generatedClassName, TypeSpec.Builder enumBuilder) {
         FileSpec.builder(generatedClassName.packageName, generatedClassName.simpleName)
+            .indent("    ")
```

Comment:
```suggestion
            .indent(" ".repeat(4))
```

Would that be overkill? 😅 It juste seems clearer to me how many spaces there are if I see the number `4`.

Reply:
Oh damn, this is groovy!! I thought it was kotlin 🤭